source/manuals/programming-languages/nodejs/style-guide.html.md.erb

```diff
@@ -0,0 +1,119 @@
+---
+title: GDS Node.js Style Guide
+---
+
+# <%= current_page.data.title %>
+
+## General style
+
+As with all JavaScript at GDS, for Node.js applications we use the [standard] style.
+
+## Use strict mode
+
+[Strict mode] should be enabled for Node.js modules, as in the browser this prevents polluting the global scope (in
+Node.js this would pollute the `global` object, rather than the `window` object).
+
+## ES2015+
+
+Because with Node.js you control the version of the interpreter you're code is running on,
+you have access to many the latest features of JavaScript, such as:
+
+- [let] and [const]
+- [Arrow functions]
+- [Array.prototype] methods like `Array.map()`, `Array.find()` and `Array.values()`
+- The [spread operator]
+
+However use caution when using cutting edge JavaScript features and be sure to read the [documentation].
+
+A full list of what features are supported in each version of Node.js can be found at [Node Green].
+
+
+### let and const
+
+For Node.js code, be explicit about variable mutability using [const] and [let]:
+
+```
+const foo // GOOD: explicitly immutable variable declaration
+let bar // GOOD: expicitly mutable variable declaration
+var baz // BAD: not explicit about mutability of variable
+```
+
+This makes code easier to understand, as well as providing a feedback loop if a [const] variable is inadvertently reassigned.
+
+### Promises
+
+For Node.js applications, we use the native `Promise` implementation available in Node.js over alternatives like `q` or `bluebird`.
+This allows for a consistent Promise implementation between projects.
+
+Promise rejections should, as a general rule, reject with an instance of `Error` or one of it's subclasses. More on [Node.js Errors].
+
+```
+'use strict'
+
+// NPM Dependencies
+const request = require('request')
+
+function doSomethingAsync() {
+  return new Promise((resolve, reject) => {
+    request.get('https://www.example.com', (err, response, body) => {
+      if (err) {
+        reject(err)
+      } else {
+        resolve(body)
+      }
+    })
+  })
+}
+```
+
+
+## Organise your imports
+
+Import your dependencies at the top of each file, and group them into 'core', 'npm' and 'local' dependencies, like in the below example.
+This makes your imports clearer and allows others to  more easily understand and maintain cross dependencies of each file.
+
+```
+'use strict'
+
+// Core Dependencies
```

Comment:
I think these should be sentence case

Reply:
As in the below?

`// Core dependencies`

Reply:
Yeah, tiny thing but worth it if this will be replicated in every project

Reply:
Yeah sure. It's the tiny things that matter more often than not.

Will update the PR.